src/torch/1-split-encoder-classifier/split-encoder-classifier.lua

```diff
@@ -8,13 +8,14 @@
 -- to encode images.
 --
 
-local libsFolder = require('paths').thisfile('..')
-package.path = package.path .. ';' .. libsFolder .. '/0-tiefvision-commons/?.lua'
+local torchFolder = require('paths').thisfile('..')
+package.path = string.format("%s;%s/?.lua", os.getenv("LUA_PATH"), torchFolder)
```

Comment:
is LUA_PATH something torch sets up or is the user who has to do that?
if it's the latter, is it in th doc?